Nos.xcodeproj/project.pbxproj

```diff
@@ -830,6 +835,14 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
+		A3B943CD299ADF0600A15A08 /* XCRemoteSwiftPackageReference "keychain-swift" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/evgenyneu/keychain-swift.git";
```

Comment:
I see that this dependency was added but I don't see it being used anywhere. Can it be removed?

Reply:
I've cleared this out now.